src/XrdTpc/XrdTpcConfigure.cc

```diff
@@ -56,7 +56,22 @@ bool TPCHandler::Configure(const char *configfn, XrdOucEnv *myEnv)
                 Config.Close();
                 return false;
             }
-        } else if (!strcmp("tpc.timeout", val)) {
+        } else if (!strcmp("tpc.route", val)) {
+		if (!(val = Config.GetWord())) {
+                Config.Close();
+                m_log.Emsg("Config", "tpc.route value not specified");
+                return false;
+            }
+            if (!strcasecmp("fixed", val)){
```

Comment:
This is really overkill. Can you just use a flag? The route is either fixed or dynamic. So, m_route_fixed can be a bool and when true it's fixed and when false it's dynamic. You can do it the other way around (i.e. define m_route_dynamic).

Reply:
I'd prefer to have a config variable that has a value assigned, so I'd expect the config file to look like this:
`tpc.fixed_route true`
but that's pretty much the same (or very similar ) code to what it's currently done. Are you asking me to change this to assume 'true' as long as the variable exists ?

Reply:
No, I was simply asking not to do constant string compares for something that can be decided by a unitary value. You can use bools and logically use the variable name to indicate what the bool value means (that was my example) or you can use an enum if you find that a variable name/value paradigm is not to your liking (though that would also be overkill given that the enum would have two values of which only one would be tested)